lib/Twig/Node/Expression/Call.php

```diff
@@ -239,9 +239,10 @@ private function getCallableParameters($callable, $isVariadic)
             if ($argument && $argument->isArray() && $argument->isDefaultValueAvailable() && array() === $argument->getDefaultValue()) {
                 array_pop($parameters);
             } else {
-                $callableName = $r->name;
-                if ($r->getDeclaringClass()) {
-                    $callableName = $r->getDeclaringClass()->name.'::'.$callableName;
+                if ($r instanceof ReflectionMethod) {
+                    $callableName = $r->class.'::'.$r->name;
+                } else {
+                    $callableName = $r->name;
```

Comment:
yes there is, if `$r` is not a `ReflectionMethod` calling `getDeclaringClass` on it will result in the error reported